src/interface/src/app/treatments/treatment-project-area/treatment-project-area.component.scss

```diff
@@ -0,0 +1,4 @@
+sg-opacity-slider {
+  display: block;
```

Comment:
i might just add this to the component, but I'll doublecheck how it behaves on the tabbed view.